### ISSUE TITLE
How to build TPC-H Tools for MySQL

### ISSUE THREAD

**Issue by AntonioZC666** [osdldbt/dbt3#2]
Hello, I met some problems while building TPC-H Tools for MySQL. I refered to the INSTALL.rst and user's guide. The user's guide says dbt3 supports MySQL but it didn't work when I want to use `dbt3-build-dbgen`. 

The  command line is following:
`./dbt3-build-dbgen --patch-dir=/home/czhang/mysql-dbt3/share/dbt3/patches --query-dir=/home/czhang/mysql-dbt3/share/dbt3/queries mysql /home/czhang/mysql-tpch`

But the ERROR is reported: `ERROR: Unhandled DBMS: mysql`

![image](https://github.com/osdldbt/dbt3/assets/61419349/61bf2212-aef2-4181-8031-a486abc3c7cc)

dbt3 doesn't seem to natively support MySQL, so I have two questions I'd like to get help:
1. How do I modify the dbt3's scripts to make it work with MySQL? Is there a more detailed user's guide?
2. Where should the extracted TPC-H-tool folder be placed? Does it have to be somewhere in the dbt3 folder? Or is it anywhere?

Thank you for your help very much!

**Comment by markwkm** [comment]
The MySQL support needs a bit of TLC.

The first patch to start figuring out how much starts with:

```diff
diff --git a/scripts/dbt3-build-dbgen.in b/scripts/dbt3-build-dbgen.in
index cd947b5..e33a7b5 100755
--- a/scripts/dbt3-build-dbgen.in
+++ b/scripts/dbt3-build-dbgen.in
@@ -119,6 +119,9 @@ done < "${SERIESFILE}"
 cd dbgen || (echo "ERROR: cannot enter directory ${DBGENDIR}/dbgen" && exit 1)

 case "${DBMS}" in
+	(mysql)
+		DATABASE="mysql"
+		;;
 	(pgsql)
 		DATABASE="postgresql"
 		;;
```

I should be able to help in understanding how the scripts are supposed to work, but we'll need someone who understands MySQL administration and SQL specific syntax.

**Comment by markwkm** [comment]
I took an initial stab at fixing some of the build and run scripts, available in the v2.0.10 release.  I'm pretty sure I didn't fix all the possible usage cases.  I'm going to go ahead and close this issue but we can keep opening more as specific issues come up.
